Sage/carbon
# 1766: Fixed warning icon overlapping text in input field.

# Description

Added padding and handled text overflow to avoid warning icon overlapping the text inside input fields.

# TODO

# Related Issues / Pull Requests

https://github.com/Sage/carbon/issues/1766

# Screenshots / Gifs

In-app QA:
![screen shot 2018-06-22 at 15 47 21](https://user-images.githubusercontent.com/32830770/41783042-caf573f6-7633-11e8-837d-55742146fda7.png)

Carbon website QA:
<img width="1244" alt="screen shot 2018-07-17 at 10 28 23" src="https://user-images.githubusercontent.com/32830770/42810166-2ec412aa-89af-11e8-8633-857cd6c5d1ee.png">
<img width="1311" alt="screen shot 2018-07-17 at 10 28 39" src="https://user-images.githubusercontent.com/32830770/42810167-2ee13ede-89af-11e8-87a4-2bc14ed5ec8a.png">
<img width="1316" alt="screen shot 2018-07-17 at 10 29 21" src="https://user-images.githubusercontent.com/32830770/42810168-2f030c1c-89af-11e8-8500-c8fedeb69097.png">


# Testing Instructions

Verify input fields that will use icons such as the warning icon work as expected.
